File: vue/src/services/generatorService.ts

const inputRange = '0123456789abcdefghijjklmnopqrstuvwxyz';
const inputChars = inputRange.split('');
const upperLimit = inputChars.length;

export const generatorService = {
    generate,
};

function generate(
    length: number,
) {
    let result = '';
    for (let i = 0; i < length; i++) {
        const random = Math.floor(
            Math.random() * upperLimit
        );
        result += inputChars[
            random
        ];
    }
    return result;
}

File: api.net/Services/GeneratorService.cs
namespace api.net.Services
{
    using api.net.Interfaces;
    using System;
    using System.Linq;
    public class GeneratorService :
        IGeneratorService
    {
        readonly Random Random =
            new Random();
        readonly string[] Range =
            "0123456789abcdefghijjklmnopqrstuvwxyz"
            .ToCharArray()
            .Select(x => x.ToString())
            .ToArray();
        public string Generate(
            int length
        )
        {
            var result = "";
            for (var i = 0; i < length; i++)
            {
                var n = Random.Next(
                    0,
                    Range.Length
                );
                result += Range[n];
            }
            return result;
        }
    }
}
